app/test_main.py

```diff
@@ -2,0 +1,11 @@
+from app.main import can_access_google_page
+
+
+def test_can_acces_google_page_for_acessible() -> None:
+    url = "https://example.com"
+    assert can_access_google_page(url) == "Accessible"
+
+
+def test_can_acces_google_page_for_not_acessible() -> None:
```

Comment:
The function name `test_can_acces_google_page_for_not_acessible` contains typos. It should be `test_can_access_google_page_for_not_accessible`. Consider using a more descriptive name, such as `test_inaccessible_url_without_internet_connection`.